### ISSUE TITLE
(LDAP Integration) Realize an agreed upon 'User Profile' (aka LDAP schema)

### ISSUE THREAD

**Issue by hmitsch** [mozilla-iam/iam-project-backlog#49]

**Comment by gdestuynder** [comment]
Work started at https://github.com/mozilla-iam/ldap-postgres-reference/tree/master/doc

**Comment by gdestuynder** [comment]
Current notes:
https://github.com/gdestuynder/ldap-postgres-reference/blob/master/doc/User_profile.json#L85 is to be updated by auth0 on creation/update, this can be done through a rule, or via CIS

https://github.com/gdestuynder/ldap-postgres-reference/blob/master/doc/User_profile.json#L108 is a little odd because its automatically generated. Has to be done via CIS. Could make this better.

**Comment by gdestuynder** [comment]
Review needed:

- [ ] @jdow 
- [x] @andrewkrug 
- [ ] @gene1wood 
- [x] @flamingspaz 
- [ ] @akatsoulas 
- [x] @johngian

**Comment by andrewkrug** [comment]
@gdestuynder is this still waiting on any specific review from me?  I think the proposed profile is sufficient as a starting point.

**Comment by gdestuynder** [comment]
nope checkbox checked is good :)

**Comment by gdestuynder** [comment]
Made this a new PR https://github.com/mozilla-iam/ldap-postgres-reference/pull/4
Includes all feedback/etc so far. Awaiting @johngian feedback on user stories/data for potentially missing Mozillians fields such as Geoloc (please commit them as new user stories, it's the easiest i think, with rationale of why we want em)

Nevertheless I think this is sufficient to get started on CIS code.

**Comment by gdestuynder** [comment]
PR of Integration of the user profile to auth0 (initial version at least, but it's functional and is one of the required pieces)

https://github.com/mozilla-iam/auth0-deploy/pull/108

**Comment by gdestuynder** [comment]
Should add SAML mapping in auth0 for the profile

**Comment by gene1wood** [comment]
@gdestuynder What specifically are you looking for review of in [this comment](https://github.com/mozilla/iam-project-backlog/issues/49#issuecomment-288237128)

**Comment by gdestuynder** [comment]
the comments that were above it, but honestly its too late. we agreed on the profile and will iterate from it from now on.
This is the decided profile: https://github.com/mozilla-iam/ldap-postgres-reference/blob/master/doc/User_profile.json
And iteration will appear in http://github.com/mozilla-iam/cis

Therefore also closing this
